### PR TITLE
Add: controller for user search by email

### PR DESCRIPTION
**Pull request by lazy-god**

### CODE REVIEW

File: src/routers/api/users.js [hunk 1]
```diff
@@ -39,6 +39,33 @@ router.get('/',
   }
 )
 
+router.get('/email',
```
Comment (championswimmer) [inline src/routers/api/users.js]: this should be a get request to `/users/` only with `email=xxx@yyy.com` in the body

Comment (witty123) [reply src/routers/api/users.js]: @firefinchdev can you please look into this.

Comment (firefinchdev) [reply src/routers/api/users.js]: Will see into this soon...

Comment (firefinchdev) [reply src/routers/api/users.js]: @championswimmer , in that case this pr is unnecessary. user search by email was already implemented in `findAllUsersWithFilter`

Comment (championswimmer) [reply src/routers/api/users.js]: @himankbhalla @witty123 
Already exists. Closing this.

**Pull request closed without merging**